src/config/types.ts

```diff
@@ -128,8 +128,94 @@ export interface ExpectOptsConfig {
     interval: number;
 }
 
-export interface MochaOpts extends Omit<Mocha.MochaOptions, "ui"> {
```

Comment:
Remove `Mocha.MochaOptions` in order to not get conflicts in `it`, `describe` etc. Reproduced when service has `@types/mocha` module (can be installed as dependencies from some other module).